db/migrate/20240111192555_create_posts.rb

```diff
@@ -1,11 +1,11 @@
 class CreatePosts < ActiveRecord::Migration[7.1]
   def change
     create_table :posts do |t|
-      t.references :author, null: false, foreign_key: true
+      t.references :author, null: false, foreign_key: { to_table: 'users' }
       t.string :title
       t.text :text
-      t.datetime :created_at
-      t.datetime :updated_at
+      t.datetime :createdAt
+      t.datetime :updatedAt
```

Comment:
- [ ] **[OPTIONAL]** I would suggest you remove these columns. Besides, the fact they do not follow snake case rule for table columns, there is no use for these `updateAt` and `createdAt` columns. The `t.timestamps` takes care of the `updated_at` and `created_at` columns that we need.